src/gui/Gui.cpp

```diff
@@ -3,84 +3,313 @@
 /// Copyright (c) 2021-present Anonymous275 read the LICENSE file for more info.
 ///
 
-#define _CRT_SECURE_NO_WARNINGS
 #include <wx/wxprec.h>
-#include <set>
 #ifndef WX_PRECOMP
-#include <wx/animate.h>
-#include <wx/mstream.h>
+#include <wx/dc.h>
+#include <wx/dcbuffer.h>
+#include <wx/fs_inet.h>
+#include <wx/graphics.h>
+#include <wx/hyperlink.h>
+#include <wx/sizer.h>
+#include <wx/statline.h>
+#include <wx/webview.h>
 #include <wx/wx.h>
-#include "gifs.h"
+#include <wx/wxhtml.h.>
 
+#include "Launcher.h"
+#include "Logger.h"
 #endif
+
+/////////// Inherit App class ///////////
 class MyApp : public wxApp {
    public:
-   virtual bool OnInit();
+   bool OnInit() override;
+};
+
+/////////// MainFrame class ///////////
+class MyMainFrame : public wxFrame {
+   public:
+   MyMainFrame();
+
+
+   private:
+   // Here you put the frame functions:
+   bool DarkMode = wxSystemSettings::GetAppearance().IsDark();
+   void OnClickAccount(wxCommandEvent& event);
+   void OnClickSettings(wxCommandEvent& event);
+   void OnClickLaunch(wxCommandEvent& event);
+   void OnClickLogo(wxCommandEvent& event);
+   wxDECLARE_EVENT_TABLE();
+
+};
+
+/////////// AccountFrame class ///////////
+class MyAccountFrame : public wxFrame {
+   public:
+   MyAccountFrame();
+
+   private:
+   // Here you put the frame functions:
+   bool DarkMode = wxSystemSettings::GetAppearance().IsDark();
 };
-class MyFrame : public wxFrame {
+
+/////////// SettingsFrame class ///////////
+class MySettingsFrame : public wxFrame {
    public:
-   MyFrame();
+   MySettingsFrame();
 
    private:
-   void OnHello(wxCommandEvent& event);
-   void OnExit(wxCommandEvent& event);
-   void OnAbout(wxCommandEvent& event);
-   static wxSize FixedSize;
+   // Here you put the frame functions:
+   bool DarkMode = wxSystemSettings::GetAppearance().IsDark();
 };
+
+
 enum { ID_Hello = 1 };
-wxSize MyFrame::FixedSize(370, 400);
 
+
+/////////// Event Table ///////////
+wxBEGIN_EVENT_TABLE(MyMainFrame, wxFrame)
+   EVT_BUTTON(39, MyMainFrame::OnClickAccount)
+   EVT_BUTTON(40, MyMainFrame::OnClickSettings)
+   EVT_BUTTON(41, MyMainFrame::OnClickLaunch)
+   EVT_BUTTON(42, MyMainFrame::OnClickLogo)
+wxEND_EVENT_TABLE()
+
+
+
+/////////// OnInit function to show frame ///////////
 bool MyApp::OnInit() {
-   auto* frame = new MyFrame();
-   frame->Show(true);
+   auto* MainFrame = new MyMainFrame();
+   MainFrame->SetIcon(wxIcon("BeamMP.png",wxBITMAP_TYPE_PNG));
```

Comment:
Can't find this in the repo, has this file been committed anywhere?

Reply:
"BeamMP.png" and the other image files

Reply:
The icons will be replaced from getting them locally to a link later on. But we are prioritizing other work right now.